odl/tomo/geometry/conebeam.py

```diff
@@ -639,6 +716,11 @@ def __init__(self, apart, dpart, src_radius, det_radius,
             beam geometry.
         axis : `array-like`, shape ``(3,)``, optional
             Vector defining the fixed rotation axis of this geometry.
+        src_shift_func : a function of (angle), optional
+            Should return a 3 dimensional vector representing a shift
+            relative to the default source position. Vector elements
+            represent shifts along the following directions:
+            det_to_src, tangent to the rotation, rotation axis.
```

Comment:
For helical geometries the tangent to the rotation and the rotation axis are not orthogonal. Is this intended?

Reply:
These 3 vectors must be orthogonal vector. The second vector would be tangent to the rotation if pitch was 0. Do you have any suggestions how to call it?

Reply:
I'd just be a bit verbose here and explain what you mean.